test/test_datasources.py

```diff
@@ -166,3 +164,75 @@ def test_find_agg_success(self):
                 aggfile.write("doesnt matter")
             agg_path_found = find_aggregation_netcdf_gw(nc_path)
             self.assertEqual(agg_path, agg_path_found)
+
+
+def test_get_timestamps_none(netcdf_groundwater_ds):
+    timestamps = netcdf_groundwater_ds.get_timestamps()
+    assert timestamps.shape == (32,)
+    assert timestamps[-1] == 1863.5643704609731
```

Comment:
In principe liever maar één assert per functie.

Soms kan het niet anders, hoor.

Ik heb al meerdere keren meegemaakt dat ik een foutje in m'n code eerder had gezien als ik een test gesplitst had. Als de eerste assert faalt, zie je de foutmelding van de tweede niet meer, terwijl die soms duidelijker is dan de eerste foutmelding.

Reply:
Ja idd! 

Ik ben overigens niet helemaal tevreden hoe ik dit nu getest wordt maar weet niet helemaal hoe dit beter kan. Momenteel is de test heel erg afhankelijk van de exacte data in de `test/data/testmodel/v2_bergermeer/results_3di.nc`. Deze exacte getallen moeten we weer opnieuw invullen als we dit bestand vervangen met een nieuwer resultaten bestand. 

Dat we een keer nieuwere resultaten bestanden nodig hebben is waarschijnlijk, want als er nieuwe features in 3Di (resultaten) komen, moet de plugin daar ook mee kunnen omgaan. Dus moet het getest worden.